src/main/webapp/style.css

```diff
@@ -10,15 +10,26 @@ body {
 
 .heading-font {
   font-family: 'Cinzel', serif;
-  color: rgb(237, 243, 121);
+  color: rgba(0,0,0, 0.4);
   font-size: 100px;
   text-align: center;
   
 }
 
 .body-font {
```

Comment:
I'd suggest renaming the class -- this looks like it handles more than just the font.

If this is the standard font, consider moving the font-related stuff to

`body {

}`

A lot of CSS properties are inherited from the parent (it's the "cascade" in the part of the CSS name)

Reply:
okay that makes sense - I'll look into if we may use any other fonts and change accordingly.